WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StatsTimeRangeBarView.swift

```diff
@@ -62,6 +71,14 @@ private extension StatsTimeRangeBarView {
         }
         stackView.addArrangedSubview(button)
     }
+
+    func configureSelectedDateLabel() {
+        selectedDateLabel.font = Constants.labelFont
+        selectedDateLabel.textColor = Constants.labelColor
+        selectedDateLabel.textAlignment = Constants.labelTextAlignment
+        selectedDateLabel.isHidden = true
```

Comment:
If you apply the change sugggested above, you can remove this `isHidden` setter.